Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -630,10 +631,12 @@ public bool UnregisterPointer(IMixedRealityPointer pointer)
 
                 if (!objectIsStillFocusedByOtherPointer)
                 {
+                    MixedRealityToolkit.InputSystem?.RaisePreFocusChanged(pointer, unfocusedObject, null);
```

Comment:
We should raise pre focus changed/focus changed regardless of the value of objectIsStillFocusedByOtherPointer